### PR TITLE
internal/params: schedule HIP28 and val code fix

### PR DESCRIPTION
**Pull request by MaxMustermann2**
2023-07-20 05:51:07+00:00

### CODE REVIEW

**Review by ONECasey — APPROVED** [review]

**Review by GheisMohammadi — APPROVED** [review]

**Comment by ONECasey** [comment]
Lets wait until @sophoah approves to merge.

**Review by sophoah — APPROVED** [review]

**Comment by sophoah** [comment]
I am going update to merge this to dev and include that in 2023.2.4

**Comment by MaxMustermann2** [comment]
Testnet output:

Contract from #4413 deployed on `0xE890b1e8C7f6bD0fF82B586393b9dC67635e4c28`. Check validator address `0x29c2eC57803f8b695f02613E5FA1749c165c5057`. 
```python
>>> contract.extCodeCopy( accounts[ 1 ].address )
0x
>>> contract.extCodeCopyCall( accounts[ 1 ].address )
0x
>>> contract.extCodeSize( accounts[ 1 ].address )
0
>>> contract.extCodeSizeCall( accounts[ 1 ].address )
0
>>> web3.eth.getCode( acconts[ 1 ].address )
HexBytes('0xTrunacted')
```

**Pull request merged**
